src/migrations/tasks/20230327102038_accept-twu-terms.ts

```diff
@@ -1,30 +1,19 @@
-import { generateUuid } from "back-end/lib";
 import { makeDomainLogger } from "back-end/lib/logger";
 import { console as consoleAdapter } from "back-end/lib/logger/adapters";
 import Knex from "knex";
 
 const logger = makeDomainLogger(consoleAdapter, "migrations");
 
-const slug = "team-with-us-terms-and-conditions";
-
 export async function up(connection: Knex): Promise<void> {
   await connection.schema.alterTable("organizations", (table) => {
     table.timestamp("acceptedTWUTerms").nullable();
   });
   logger.info("Modified organizations table.");
 
-  // Add TWU terms content initial value
-  const now = new Date();
-  const body = "Initial version";
-  const id = generateUuid();
-  await connection("content").insert({ id, createdAt: now, slug, fixed: true });
-  await connection("contentVersions").insert({
-    id: 1,
-    title: slug,
-    body,
-    createdAt: now,
-    contentId: id
```

Comment:
Is this change intentional?

Reply:
yup, @bdolor already covered it in a different migration apparently.